### PR TITLE
Added interpolation methods

### PR DESCRIPTION
**Pull request by bchevalier**
Tina is more stable, so the warning in the readme got smaller.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,22 +18,32 @@ A comprehensive, high performance, easy to use, open source animation library in
...
 
 Note: Do not hesitate to contribute by reporting issues or by submitting your own components and interpolations.
 
+Warning: Still in beta version!
+
 ## How to use
 
-Include tina's build in your html using either the [minified library](https://raw.githubusercontent.com/Wizcorp/tina/master/build/tina.min.js) or the [unminified version](https://raw.githubusercontent.com/Wizcorp/tina/master/build/tina.js).
+### In a browser
+Include TINA's build in your html using either the [minified library](https://raw.githubusercontent.com/Wizcorp/tina/master/build/tina.min.js) or the [unminified version](https://raw.githubusercontent.com/Wizcorp/tina/master/build/tina.js).
```
Comment (ronkorving) [inline README.md:31]: Or use browserify :)

File: README.md [hunk 2]
```diff
@@ -19,22 +18,32 @@ A comprehensive, high performance, easy to use, open source animation library in
...
+```javascript
+var TINA = require('tina');
+```
+
 ## API
 
-Existing playable components are: Tween, NestedTween, Timeline, Sequence, Delay, Timer, Ticker (Recorder coming soon).
-The following is a non-exhaustive list of possibilities offered by the API.
+Existing playable components are: **Tween, NestedTween, Timeline, Sequence, Delay, Timer, Ticker** (Recorder and CSS Tween coming soon).
+The following is a non-exhaustive list of possibilities offered by the TINA.
```
Comment (ronkorving) [inline README.md:46]: "the TINA"

File: src/easing.js [hunk 3]
```diff
@@ -25,68 +25,121 @@ var TWO_PI      = Math.PI * 2;
...
+
+exports.cubicOut = function (t) {
+	return 3 * t - 3 * t * t + t * t * t;
+};
+
+exports.cubicInOut = function (t) {
+	if (t < 0.5) {
+		return 4 * t * t * t;
+	} else {
+		return 4 * (3 * t - 3 * t * t + t * t * t) - 3;
```
Comment (ronkorving) [inline src/easing.js:93]: 3 \* t - 3 \* t ...
Don't they cancel each other out?

File: src/interpolation.js [hunk 4]
```diff
@@ -101,7 +101,7 @@ exports.colorRGBToString = function(t, a, b) {
 	return 'rgb(' + cr.toString(16) + ',' + cg.toString(16) + ',' + cb.toString(16) + ')';
```
Comment (ronkorving) [inline src/interpolation.js:101]: These are not hex, they are decimal 0-255.

File: src/interpolation.js [hunk 5]
```diff
@@ -101,7 +101,7 @@ exports.colorRGBToString = function(t, a, b) {
 	return 'rgb(' + cr.toString(16) + ',' + cg.toString(16) + ',' + cb.toString(16) + ')';
 };
 
-exports.colorRGBAToString = function(t, a, b) {
+exports.colorRGBAToString = function (t, a, b) {
 	var cr = Math.round(a.r * (1 - t) + b.r * t);
 	var cg = Math.round(a.g * (1 - t) + b.g * t);
 	var cb = Math.round(a.b * (1 - t) + b.b * t);
```
Comment (ronkorving) [inline src/interpolation.js:107]: rgb are not hex, they are decimal 0-255.
a is 0-1

File: src/interpolation.js [hunk 6]
```diff
@@ -93,7 +93,7 @@ exports.colorRGBToHexa = function(t, a, b) {
 	return '#' + cr.toString(16) + cg.toString(16) + cb.toString(16);
```
Comment (ronkorving) [inline src/interpolation.js:93]: I think this breaks on low numbers, no? `3..toString(16)` will return `"3"`, not `"03"`.

File: README.md (hunk 2 above)
Comment (bchevalier) [inline README.md:46]: yes, the only one

Comment (ronkorving) [inline README.md:46]: lol, it used to say "the API", where "the" made sense.. but hey, you go for it ;)

**Pull request merged**